### ISSUE TITLE
Section 1 Feedback

### ISSUE THREAD

**Issue by michelnev** [w3c/webmediaguidelines#46]
I have a few suggestions for section 1 of the document.  

1. In Scope, suggest changing "(applications that _has_ been developed for use on a particular platform or device)" to "(applications that **have** been developed for use on a particular platform or device)"

2.  In Glossary of Terms, suggest changing "In order to provide a common language to build this document and communicate concepts to the end reader, we are providing a Glossary of Terms to this guideline document." to something like "The following lexicon provides the common language used to build this document and communicate concepts to the end reader."

3. In Glossary of Terms: Codec, suggest rewording "A codec is the algorithm used to capture analog video or audio in digital form."  In our context, a codec is an algorithm defining the compression and decompression of digital video or audio, not a way to capture analog A/V.

4. In Glossary of Terms: Bit Rate,  suggest changing "and can be constant _and_ variable" to "and can be constant **or** variable."

5. In Glossary of Terms: EME, because of the recent announcement, please update "Encrypted Media Extensions (EME) is a _proposed_ W3C specification" to "Encrypted Media Extensions (EME) is a **recommended** W3C specification".

6. In Glossary of Terms: Format, suggest removing the comma right after "container format".

7. In Glossary of Terms: manifest, suggest capitalizing manifest to match all the other terms.

8. In Glossary of Terms: Rendition, please change "A specific video and audio stream for a target quality level or bitrate in _a_ adaptive streaming set or manifest" to "A specific video and audio stream for a target quality level or bitrate in **an** adaptive streaming set or manifest"

9. In Glossary of Terms: Self-hosted player, suggest using camel case for the term to match the rest of the glossary, i.e. "Self-Hosted Player."  Additionally, the "Note: we do not recommend this as the player updates every 4-6 weeks." sentence should be removed because it reads like marketing material.

10. In Glossary of Terms: Streaming, suggest change "Streaming of media content is typically the delivery of media assets over Internet protocols such as HTTP or RTMP." to something like "Streaming of media content is typically the delivery of media assets using Internet protocols such as HTTP or RTMP."  Removing "typically" might be useful, since TCP/IP streaming is our focus.

11. In Glossary of Terms: Encoder and Transcoder, I don't consider encoding a synonym for transcoding.  I think transcoding is the processes of converting from one encoded format to another.  Encoding is closely aligned with the existing description.

12. In Glossary of Terms: URL Signing, I'd suggest removing the entire last sentence.  It is currently awkwardly written.   "This ensures that only links generated by the customer can be used to access their content and that if those links are used elsewhere, the use will expire at the expiration time."  It could also be reworded.

**Comment by joelkorpi** [comment]
Changes made and merged into master. PR #63
